client/commands/v2/pysa_server.py

```diff
@@ -11,7 +11,12 @@
 
 import asyncio
 import logging
+from os.path import relpath
+from pathlib import Path
+from typing import List, Optional, Sequence, Dict
 
+from ....api import query, connection as api_connection
+from ....api.connection import PyreQueryError
```

Comment:
@grievejia had some feedback about using `api` on Sarthak's PR:
https://github.com/facebook/pyre-check/pull/433/files#r704607152

Could you take a look at his suggested alternative?

Reply:
Hmm, it looks like the new logic for querying `types` is also in the `api` package - would you recommend refactoring them into `query.py` or a separate file in `client.commands.v2`? Lmk if you have any other suggestions for going about this!

Reply:
My first thought would be a separate file, but I don't feel to strongly about it.